src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,28 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  visibleModal: Todo;
+  setVisibleModal: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  visibleModal, setVisibleModal,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(visibleModal.userId).then(setUser);
+  }, []);
```

Comment:
The useEffect hook's dependency array is empty, which means it will only run once after the initial render. However, you are using 'visibleModal.userId' inside the hook, so this should be included in the dependency array. This way the hook will run again whenever 'visibleModal.userId' changes.